internal/util/util.go

```diff
@@ -68,19 +73,39 @@ func ExecuteCommand(cmd *exec.Cmd, pathPrefix string, envars []string) {
 
 	err := cmd.Start()
 	if err != nil {
-		log.Fatalf("Starting command failed: %v\n", err)
+		return err
 	}
 
 	go func() {
 		_, errStdout = io.Copy(os.Stdout, stdoutPipe)
 	}()
 	go func() {
-		_, errStderr = io.Copy(os.Stderr, stderrPipe)
+		strErrStreams := []io.Writer{errContent}
```

Comment:
is this supposed to be `stdErrStreams`?

Reply:
sorry yeah, should have paid more attention with all these typos